src/recap

```diff
@@ -283,49 +282,47 @@ print_mysql_innodb_status() {
                                     SHOW VARIABLES LIKE 'tmpdir'; \
                                     SHOW ENGINE INNODB STATUS;" 2>/dev/null \
                 | head -n 2 ) )
-  PID_FILE=${MYVALS[1]%/}
-  TMPDIR=${MYVALS[3]%/}
+  PID_FILE="${MYVALS[1]%/}"
+  TMPDIR="${MYVALS[3]%/}"
 
   # Next, we grab a list of descriptors in the tmpdir:
   for name in "/proc/"$(cat ${PID_FILE})"/fd/"*; do
     # We know that InnoDB's temporary files are always in the mysql tmpdir
     # and start with the string 'ib' followed by a randomly generated series
     # of characters.  Now we can compare the canonicalized path of fd to the
     # tmpdir/ib* pattern:
-    [[ $(readlink -f "${name}") == "${TMPDIR}"/ib* ]] &&
-    # If any files match that pattern, we see if the first line in the file
-    # starts with "===", which is the case in the particular InnoDB file we
-    # care about:
-      head -c8 "${name}" | grep -q '^====' &&
-    # Finally, print out the information and append it to the mysql file:
-    cat "${name}"  >> "${LOGFILE}"
+    if [[ "$(readlink -f "${name}")" == "${TMPDIR}"/ib* ]]; then
+      # If any files match that pattern, we see if the first line in the file
+      # starts with "===", which is the case in the particular InnoDB file we
+      # care about:
+      head -c8 "${name}" | grep -q '^====' && cat "${name}"  >> "${LOGFILE}"
+    fi
   done
 }
 
 # print the output of "mysqladmin processlist" to the mysql file
 print_mysql_procs() {
-  local LOGFILE=$1
+  local LOGFILE="$1"
   # We only support a single DOTMYDOTCNF for MySQL when running Plesk
   local PLESK_FILE="/etc/psa/.psa.shadow"
   if [[ -r "${PLESK_FILE}" ]]; then
     echo "MySQL processes" >> "${LOGFILE}"
     if [[ ${MYSQL_PROCESS_LIST} == "table" ]]; then
-      mysqladmin -uadmin -p$( cat "${PLEASK_FILE}" ) -v processlist >> "${LOGFILE}"
+      mysqladmin -uadmin -p$( cat "${PLESK_FILE}" ) -v processlist >> "${LOGFILE}"
     fi
     if [[ ${MYSQL_PROCESS_LIST} == "vertical" ]]; then
       mysql -uadmin -p$( cat "${PLESK_FILE}" ) -e "show full processlist\G" >> "${LOGFILE}"
     fi
     print_blankline "${LOGFILE}"
   else
     for cnf in $( sed -e 's/,/ /g' <<<"${DOTMYDOTCNF}" ); do
-      if [[ -f "${cnf}" ]]; then
+      if [[ -r "${cnf}" ]]; then
         echo "MySQL ${cnf} processes" >> "${LOGFILE}"
         if [[ ${MYSQL_PROCESS_LIST} == "table" ]]; then
           mysqladmin --defaults-extra-file="${cnf}" -v processlist >> "${LOGFILE}"
-        fi
-        if [[ ${MYSQL_PROCESS_LIST} == "vertical" ]]; then
+        elif [[ ${MYSQL_PROCESS_LIST} == "vertical" ]]; then
           # This doesnt seem to work right
-          mysqladmin --defaults-extra-file="${cnf}" --vertical processlist
+          mysqladmin --defaults-extra-file="${cnf}" -e "show full processlist\G"
```

Comment:
Still broken: s/mysqladmin/mysql/
That being said, you fixed it for real in https://github.com/rackerlabs/recap/pull/78/commits/d1d27cdbce58069d099e902fca8c33189ab594a8, so nevermind.